### PR TITLE
Fixed typos + minor refact

### PR DESCRIPTION
**Pull request by xylographe**
The two if-expressions in `src/Forms/DvdSubRipChooseLanguage.cs` look like 'always true'. But, they're probably there for a reason.

### CODE REVIEW

File: src/Forms/DvdSubRipChooseLanguage.cs [hunk 1]
```diff
@@ -143,7 +143,7 @@ private void ComboBoxLanguagesSelectedIndexChanged(object sender, EventArgs e)
 
         private void ButtonOkClick(object sender, EventArgs e)
         {
-            if (listBox1.Items.Count > -1)
+            if (listBox1.Items.Count >= 0)
```
Comment (xylographe) [inline src/Forms/DvdSubRipChooseLanguage.cs]: always true?

File: src/Forms/DvdSubRipChooseLanguage.cs [hunk 2]
```diff
@@ -175,7 +175,7 @@ private void DvdSubRipShowSubtitles_KeyDown(object sender, KeyEventArgs e)
 
         private void buttonSaveAs_Click(object sender, EventArgs e)
         {
-            if (listBox1.Items.Count > -1)
+            if (listBox1.Items.Count >= 0)
```
Comment (xylographe) [inline src/Forms/DvdSubRipChooseLanguage.cs]: always true?

File: src/Forms/Beamer.cs [hunk 3]
```diff
@@ -288,7 +288,7 @@ private Bitmap GenerateImageFromTextWithStyle(string text)
                     sb = new StringBuilder();
 
                     int endIndex = text.Substring(i).IndexOf('>');
-                    if (endIndex == -1)
```
Comment (ivandrofly) [inline src/Forms/Beamer.cs]: I think this isn't need since the method is built only to return -1 or >

Comment (xylographe) [inline src/Forms/Beamer.cs]: I think, the assumption is, that comparing against 0 is more efficient than comparing against -1 ([here for example](https://github.com/SubtitleEdit/subtitleedit/commit/61d8bda6fc4a137152c81d1371b9e767b9aaab90#diff-65a9c06f48cd76263659510678e33a72R852)). Although [my own test](https://www.dropbox.com/s/recumj0rm8nxl6f/MinusOne-Zero.cs?dl=0) gave inconclusive results.

File: src/Forms/DvdSubRipChooseLanguage.cs (hunk 1 above)
Comment (ivandrofly) [inline src/Forms/DvdSubRipChooseLanguage.cs]: Nice one :1

Comment (niksedk) [inline src/Forms/DvdSubRipChooseLanguage.cs]: he, yes - I guess the if statement can be removed

File: src/Forms/DvdSubRipChooseLanguage.cs (hunk 2 above)
Comment (niksedk) [inline src/Forms/DvdSubRipChooseLanguage.cs]: Yeah...

**Pull request merged**